runners/flink/runner/src/test/java/org/apache/beam/runners/flink/streaming/TestCountingSource.java

```diff
@@ -0,0 +1,256 @@
+/*
```

Comment:
Ah, I needed it because the test requires an `UnboundedSource` that stops after emitting a certain number of elements. (But isn't a `BoundedSource`.). Maybe I can restructure the test to get around it.